image_masking.py

```diff
@@ -0,0 +1,125 @@
+import numpy
+from collections import namedtuple
+from chainer.backends import cuda
+
+Size = namedtuple('Size', ['height', 'width'])
```

Comment:
What is this variable for? I've not seen it used anywhere? Is it maybe implicitly required for the GPU code to run?

Reply:
That is a good question, I will test if the code also works without it